operator/docs/configuration.md

```diff
@@ -27,6 +29,53 @@ metadata:
     tailing-sidecar: <container-name0>:<volume-name0>:<path-to-tail0>;<volume-name1>:<path-to-tail1>;<volume-name2>:<path-to-tail2>
 ```
 
-## Examples
+Example configurations in annotations for Kubernetes resources can be found in [examples](../examples) directory.
+
+**Notice**: Only basic options can be configured in annotations, for extended configuration options please
+see [Configuration in TailingSidecarConfig](#configuration-in-tailingsidecarconfig).
+
+## Configuration in TailingSidecarConfig
+
+Configuration for Tailing Sidecar Operator can be provided through `TailingSidecarConfig` which is a Custom Resource
+used by the operator.
+
+Example definitions of `TailingSidecarConfig` are available in [samples](../config/samples) directory and Pod adjusted to this configuration
+in available [pod-with-tailing-sidecar-config.yaml](../examples/pod_with_tailing_sidecar_config.yaml).
+
+To try example configuration use following commands:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/main/operator/config/samples/tailing-sidecar_v1_tailingsidecar.yaml
+kubectl apply -f https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/main/operator/examples/pod_with_śtailing_sidecar_config.yaml
```

Comment:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/main/operator/examples/pod_with_tailing_sidecar_config.yaml
```

Reply:
Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/32aaf3e15ae19592a75529f0e4f4742c3477da20